lib/documentLoader.js

```diff
@@ -152,3 +159,21 @@ async function _pluckDidNode(did, target, didDocument) {
     filtered, {'@context': context, id: target}, {embed: '@always'});
   return Object.assign({'@context': context}, framed['@graph'][0]);
 }
+
+async function _webLoader(url) {
+  if(!url.startsWith('http')) {
+    throw new Error('NotFoundError');
+  }
+  let result;
+  try {
+    result = await axios(url);
```

Comment:
```suggestion
    result = await axios.get(url, headers: {Accept: 'application/ld+json, application/json'});
```

This may need to be changed to something similar to https://github.com/digitalbazaar/edv-client/blob/master/EdvClient.js#L423

Reply:
An `httpsAgent` may need to be provided as well. That would involve bringing in `bedrock-https-agent`.